Sources/Subs.php

```diff
@@ -16,6 +16,8 @@
 if (!defined('SMF'))
 	die('No direct access...');
 
+require_once($sourcedir . '/Class-Data_Validator.php');
+
```

Comment:
We don't call stuff like this here.

If its needed at startup, should go in the index/ssi.php with the other startup require_once calls.  If another function will always call it to start it up, that function can do the require